lint-tasks.yml

```diff
@@ -4,7 +4,12 @@ vars:
   G_LINT_VENV_DIR: "{{.BUILD_DIR}}/lint-venv"
   G_LINT_VENV_CHECKSUM_FILE: "{{.BUILD_DIR}}/lint#venv.md5"
   # Linter target dirs
-  G_CPP_LINT_DIRS: ["{{.CLP_FFI_PY_CPP_SRC_DIR}}"]
+  G_CPP_LINT_DIRS:
```

Comment:
We need to use a different variable since this variable is also used for the cpp-format tasks.